prusti-viper/src/encoder/encoder.rs

```diff
@@ -1680,6 +1687,29 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
         };
         result
     }
+
+    pub fn add_discriminant_info(&self, enum_id: String, discr_id: String, proc_def_id: ProcedureDefId) {
+        let mut map = self.discriminants_info.borrow_mut();
+        let mut previous_entries = map.get_mut(&(proc_def_id.clone(), enum_id.clone()));
+        match previous_entries {
+            None => {map.insert((proc_def_id, enum_id), vec![discr_id]);},
+            Some(v) => {
+                v.push(discr_id);
+            }
+        };
+    }
+
+    pub fn discriminants_info(&self) -> HashMap<(ProcedureDefId, String), Vec<String>> {
+        self.discriminants_info.borrow().clone()
+    }
+/*
+    pub fn get_counterexample(
+        &self, 
+        def_id: ProcedureDefId,
+        silicon_counterexample: Option<SiliconCounterexample>
+    ) -> Counterexample {
+        backtranslate(&self, def_id, silicon_counterexample);
+    }*/
```

Comment:
What is this?

Reply:
Sorry, that's a left-over from an earlier stage.